nodejs/src/wrapper.ts

```diff
@@ -25,6 +25,11 @@ import { startTracing } from '@splunk/otel';
 const logLevel = getEnv().OTEL_LOG_LEVEL
 diag.setLogger(new DiagConsoleLogger(), logLevel)
 
+// configure flush timeout
+let forceFlushTimeoutMillisEnv = process.env.OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT
+const forceFlushTimeoutMillis = parseInt(forceFlushTimeoutMillisEnv ? forceFlushTimeoutMillisEnv : "3000")
```

Comment:
This will be passed as `NaN` if the env var value doesn't resemble a number. Maybe error or warn and default in this case?

Reply:
Good idea. How about now (Nan handling) ?